src/app/public/modules/select-field/select-field.component.ts

```diff
@@ -110,9 +110,11 @@ export class SkySelectFieldComponent implements ControlValueAccessor {
   }
 
   public set value(value: any) {
-    this._value = value;
-    this.onChange(this.value);
-    this.onTouched();
+    if (value !== this._value) {
```

Comment:
Could you verify this works with arrays. Unless I'm misreading things `value` could be an array if in multi-select mode. This equality check will always fail in this case as it will check if it is actually the same array not that the contents are the same.

Reply:
Hmm, I hadn't thought about multi-select. Good catch.
Is it cheating if I just change this to `JSON.stringify(value) !== JSON.stringify(this._value)`? 😆 

I can always right a little array comparer instead